modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1110,8 +1124,13 @@ def _read(cls, filepath_or_buffer, **kwargs):
             nrows=None,
             compression=compression_infered,
         )
+        fname = partition_kwargs.pop("fname")
+        kwargs_ref = cls.put(partition_kwargs)
```

Comment:
maybe it is worth adding comments regarding why we preprocess `partition_kwargs` here and `func` inside of `_launch_tasks`?

Reply:
added